### PR TITLE
Build Debian packages for Ubuntu 21.04 Hirsute

### LINKED ISSUE #9732: Add support for Ubuntu’s Hirsute Hippo (21.04)
Hirsute is on the way to release: https://lists.ubuntu.com/archives/ubuntu-announce/2021-April/000267.html

### PR DESCRIPTION
**Pull request by callahad**
All debs build locally without error

Fixes #9732 

Signed-off-by: Dan Callahan <danc@element.io>

### Pull Request Checklist

* [ ] Pull request is based on the develop branch
* [x] Pull request includes a [changelog file](https://github.com/matrix-org/synapse/blob/master/CONTRIBUTING.md#changelog). The entry should:
  - Be a short description of your change which makes sense to users. "Fixed a bug that prevented receiving messages from other servers." instead of "Moved X method from `EventStore` to `EventWorkerStore`.".
  - Use markdown where necessary, mostly for `code blocks`.
  - End with either a period (.) or an exclamation mark (!).
  - Start with a capital letter.
* [x] Pull request includes a [sign off](https://github.com/matrix-org/synapse/blob/master/CONTRIBUTING.md#sign-off)
* [x] Code style is correct (run the [linters](https://github.com/matrix-org/synapse/blob/master/CONTRIBUTING.md#code-style))

### CODE REVIEW

File: scripts-dev/build_debian_packages
Comment (richvdh) [inline scripts-dev/build_debian_packages:25]: we should probably mention we;re dropping support for focal in the changelog.

Comment (callahad) [inline scripts-dev/build_debian_packages:25]: ...not sure how this snuck in (or out). One sec.

Comment (callahad) [reply scripts-dev/build_debian_packages:25]: That was a mistake, fixed 😬

**Review by richvdh — APPROVED** [review]

**Comment by richvdh** [comment]
needs some changes for the builder too, see https://gitlab.matrix.org/new-vector/internal/-/wikis/Synapse-Debian-Packages#adding-a-new-distribution

**Comment by callahad** [comment]
See https://github.com/matrix-org/pkg-repo-configs/pull/4

**Comment by callahad** [comment]
(I'll merge this after we merge https://github.com/matrix-org/pkg-repo-configs/pull/4)

**Pull request merged**
